styleguide.config.js

```diff
@@ -3,6 +3,9 @@ const { version } = require('./package.json');
 
 module.exports = {
   title: `React-spatial ${version}`,
+  template: {
+    favicon: 'src/images/favicon.png',
```

Comment:
I don't know if it necessary, can you use : path.join(__dirname, ' ?

Reply:
I tried with path.join but didn't work atm.